client/src/planwise/client/scenarios/views.cljs

```diff
@@ -192,8 +191,9 @@
             suggestions-layer       [:marker-layer {:points suggested-locations
                                                     :lat-fn #(get-in % [:location :lat])
                                                     :lon-fn #(get-in % [:location :lon])
-                                                    :popup-fn   #(show-suggested-provider % state)
+                                                    :popup-fn #(show-suggested-provider % state)
                                                     :icon-fn #(suggestion-icon-fn % selected-suggestion)
+                                                    :open-fn #(= % selected-suggestion)
```

Comment:
I was thinking something along the lines of adding an `:open?` attribute to each marker, but this works too. 

Ie.
```
:points (map #(assoc % :open? (= % selected-suggestion))
             suggested-locations)
```
This requires changing the `leaflet.core` code as well of course.